plugins/dac_data_manager.c

```diff
@@ -1045,11 +1046,9 @@ static GtkWidget *gui_dac_create(struct dds_dac *ddac)
 
 	gtk_alignment_set_padding(GTK_ALIGNMENT(dac_align), 5, 5, 5, 5);
 
-	gtk_table_attach(GTK_TABLE(dac_table), gui_tx_create(&ddac->tx1),
```

Comment:
this looks like it could be squashed into the previous patch ?

Reply:
ok